GATEWAY/src/mocks.ts

```diff
@@ -166,3 +166,10 @@ export class DiagnosticMock {
     return true;
   }
 }
+
+export class ViewControllerMock {
```

Comment:
you added them, so if they weren't supposed to be there I feel you should be the one removing them 😉